docs/development.rst

```diff
@@ -1122,127 +1460,177 @@ to him until directed).
+            distribution_type="g",  # gamma distributed selection coefficient
+            distribution_args=[-0.0004, 0.27],  # mean and shape of gamma distributoin
         )
         positive = stdpopsim.MutationType(
             dominance_coeff=0.5,
-            distribution_type="f",
-            distribution_args=[0.01],
+            distribution_type="f",  # fixed selection coefficient
+            distribution_args=[0.01],  # fixed value
         )
-        # Defining some example proportions of each mutation type
+
+        # The proportions of the three mutation types
         p_neutral = 0.7
         p_negative = 0.299
         p_positive = 1 - p_neutral - p_negative
 
-        # Example return statement for this example DFE with three mutation types.
         return stdpopsim.DFE(
-            id=id,
-            description=description,
-            long_description=long_description,
+            id=...,
+            description=...,
+            long_description=...,
+            citations=...,
             mutation_types=[neutral, negative, positive],
             proportions=[p_neutral, p_negative, p_positive],
-            citations=citations,
         )
 
 
     _species.add_dfe(_dfe_func_name())
 
-See the detailed documentation of :class:`stdpopsim.DFE`
-and :class:`stdpopsim.MutationType`.
-
----------------------------------------------------
-Open a QC issue
----------------------------------------------------
-
-Open a new issue requesting a quality control check on your newly implemented
-DFE model. You should structure the issue as follows:
 
-    1. **PR for new model:**
+-----------------------------------------
+Submitting a PR and reviewing a DFE model
+-----------------------------------------
 
-    2. **Original paper:**
+After you finished your implementation, and specified all the
+necessary citations,
+you should submit a pull request (PR) with your changes to the code.
+See the `GitHub workflow`_ for more details about this process.
 
-    3. **Parameter values:**
+At this point, most of your work is done.
+**You have officially joined the** ``stdpopsim`` **development team. Welcome!!**
+Your DFE model still needs to undergo review by another member
+of the development team before it is fully incorporated into ``stdpopsim``.
+This will likely require additional feedback from you,
+so, stay tuned for discussion during the review process.
 
-    4. **Potential issues:**
+The review process for DFE models is similar to that of
+new species or demographic models.
+See the `Overview of the stdpopsim review process`_.
+There is no template issue for a DFE model QC,
+so the **QC issue** is generated by
+`opening a new blank issue <https://github.com/popsim-consortium/stdpopsim/issues/new>`__
+and specifying the following information in that issue:
 
-    5. **QC'er requests:**
+1. **PR for new model:**
+2. **Original paper:**
+3. **Parameter values:**
+4. **Potential issues:**
+5. **QC'er requests:**
 
```

Comment:
I think that this requires some more explanation. What should the contributor specify in the parameter values and potential issues. What do we mean by QCer requests? Does the contributor solicit specific people to review their PR?

We should also describe what the reviewer is expected to check. Is this just done manually, or is there any blind implementation involved here?

Reply:
I agree, but we can also just make an issue to track the problem and leave off adding these details for now. The DFE QC stuff is still a bit up in the air (I think?), but doesn't really need to be clarified until v0.2.1 which will accompany the selection manuscript.

Reply:
Okay. We can keep this a bit vague for now and I'll open an issue so that we remember to add these details to the docs before the next release with all the selection stuff.